packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -55,7 +55,7 @@ export class GraphQLGenerator {
       this.queryGenerator,
       this.subscriptionByIDResolverBuilder.bind(this),
       this.subscriptionResolverBuilder.bind(this)
-    )
+    )    
```

Comment:
Note these spaces, remember to always run `check-all-the-things` before submitting the PR so the linter fixes these things

Reply:
OK I'll check it